src/screenshots/covidcast/map_preview.py

```diff
@@ -0,0 +1,127 @@
+"""Takes a screenshot of the COVIDcast map.
+
+Extraneous UI is hidden so that the map is presentable in standalone form. The
+screenshot is intended to be suitable for use in social media previews and
+custom dashboards.
+"""
+
+# standard library
+import argparse
+import time
+import urllib.parse
+
+# third party
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium import webdriver
+
+# first party
+from delphi.operations.screenshots.utils import Utils
+
+
+class MapPreview:
+
+  # URL of the COVIDcast map with predetermined signals
+  URL = 'https://covidcast.cmu.edu/?' + urllib.parse.urlencode({
+    'sensor': 'doctor-visits-smoothed_adj_cli',
+    'level': 'county',
+    'date': '20200918',
+    'signalType': 'value',
+    'encoding': 'color',
+    'mode': 'overview',
+    'region': '42003',
+  })
+
+  # width and height in pixels of the final screenshot
+  SCREENSHOT_SIZE = (1800, 900)
```

Comment:
This seems pretty large as a default, though not a big deal if this is for experimentation for now.

Reply:
turned this into a flag so that we can make different sizes as needed